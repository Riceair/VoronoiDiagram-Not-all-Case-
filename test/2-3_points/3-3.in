#三點測試 直角三角形
3
200 200
300 200
200 300