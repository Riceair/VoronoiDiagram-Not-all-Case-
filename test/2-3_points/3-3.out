P 200 200
P 200 300
P 300 200
E -62750 250 250 250
E 250 -62750 250 250
E 250 250 63250 63250
